.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test
+
+on: [push, pull_request]
```

Comment:
Can you mute this workflow?

Reply:
It's Ok, but why? I think it's better to make sure every commits tested via this workflow.

Reply:
Oh, you are right, let's keep it!
I suggest we can have a testing graph in this repo for testing instead of using the root. To support it we may need to add a new `input` parameter, like `context` or `graph_dir` etc.

Reply:
That's great! I created an issue https://github.com/pengx17/logseq-publish/issues/6 to track it.